examples/basic/fractal.js

```diff
@@ -8,7 +8,7 @@ const paths = {
 };
 
 // Create a new theme instance with custom config options
-const theme = eclTheme({ skin: 'blue' });
```

Comment:
Also, as we're completely taking over the theme branding (also with the header/footer), the `favicon.ico` can be changed as well.